data/json/vitamin.json

```diff
@@ -6,6 +6,11 @@
     "name": { "str": "Calcium" },
     "min": -48000,
     "max": 0,
+    "//": [
+      "96 units are consumed per day, with RDA of 1000mg / 96 = 10.42mg",
+      "1000mg for 19-50yr olds from https://ods.od.nih.gov/factsheets/calcium-HealthProfessional/#h2"
+    ],
+    "weight_per_unit": "10 mg",
```

Comment:
As per https://www.fda.gov/food/nutrition-facts-label/daily-value-nutrition-and-supplement-facts-labels, the RDA for Calcium is 1300mg. I am not really sure which of these sources is more trustworthy, though, so I really just wanted to throw this out here as something to think about. Technically the FDA has been updated a year later, but I'm not 100% sure regardless...

Other than that, thank you a lot for doing this!

Reply:
The source I used has a table, I'll reproduce it here:


Age | Male | Female | Pregnant | Lactating
-- | -- | -- | -- | --
0–6 months* | 200 mg | 200 mg |   |  
7–12 months* | 260 mg | 260 mg |   |  
1–3 years | 700 mg | 700 mg |   |  
4–8 years | 1,000 mg | 1,000 mg |   |  
9–13 years | 1,300 mg | 1,300 mg |   |  
14–18 years | 1,300 mg | 1,300 mg | 1,300 mg | 1,300 mg
19–50 years | 1,000 mg | 1,000 mg | 1,000 mg | 1,000 mg
51–70 years | 1,000 mg | 1,200 mg |   |  
|>70+ years | 1,200 mg | 1,200 mg |  

It looks like the FDA is just reproducing the highest  (non-pregnant/lactating) value on the table (this is how it is with the other values). I figured most DDA characters are in the 19-50 age range, so that value would be more appropriate.

Reply:
Yep, that seems reasonable to me. Apologies for the bother, and thank you again for taking the time to do this!